robot/data.py

```diff
@@ -58,10 +58,10 @@ def add_traj(self, rgbs, joint_angles, eef_poses):
         # (bs, traj_length, rgbs)
         # (bs, traj_length, joint_angles)
         # (bs, eef_poses)
-        if self.rgb_shape is None:
-            self.rgb_shape = rgbs.shape[1:]
-            self.ja_shape = joint_angles.shape[1:]
-            self.eef_shape = eef_poses.shape[1:]
+        self.rgb_shape = rgbs.shape[1:]
+        self.ja_shape = joint_angles.shape[1:]
+        self.eef_shape = eef_poses.shape[1:]
+
```

Comment:
i don't think this changes any functionality? if shapes are changing as you're adding things, something else is wrong